Makefile

```diff
@@ -1,10 +1,18 @@
 all: lint build unit
 
 OUTPUT_DIR="build/_output"
+# Set the go mod vendor flags, if they're not already set
+GO ?=go
+GOFLAGS ?=$(shell $(GO) env GOFLAGS)
+ifeq "$(findstring -mod=vendor,$(GOFLAGS))" "-mod=vendor"
+GO_MOD_FLAGS ?=
+else
+GO_MOD_FLAGS ?=-mod=vendor
```

Comment:
Please add a space after `?=`

Reply:
Let me see if that works. I am not sure...

Reply:
You should be able to as per the [docs](https://www.gnu.org/software/make/manual/make.html#index-_003f_003d).